period_search_optimization_simd/period_search/blmatrix.cpp

```diff
@@ -1,12 +1,20 @@
-/* beta, lambda rotation matrix and its derivatives
-
-   8.11.2006
-*/
-
-#include <math.h>
```

Comment:
Trigonometric Functions requires math with GCC. Ideally use #include  <cmath> and move it to globals.h